alacritty/src/display/mod.rs

```diff
@@ -582,13 +500,14 @@ impl Display {
         match config.window.startup_mode {
             #[cfg(target_os = "macos")]
             StartupMode::SimpleFullscreen => window.set_simple_fullscreen(true),
-            #[cfg(not(target_os = "macos"))]
-            StartupMode::Maximized if is_x11 => window.set_maximized(true),
+            #[cfg(not(any(target_os = "macos", windows)))]
+            StartupMode::Maximized if !is_wayland => window.set_maximized(true),
             _ => (),
         }
 
         let hint_state = HintState::new(config.hints.alphabet());
-        let is_damage_supported = window.swap_buffers_with_damage_supported();
+
+        let is_damage_supported = is_wayland;
```

Comment:
`is_wayland` is shorter than `is_damage_supported`. So might as well check for `is_wayland` directly?

If another platform adds support in the future, we can always change it again.

Reply:
Seems fine.